### PR TITLE
adding algo info and cleaning up for standards

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Added the list of supported algorithms for inbound SAML, and simplified the topic so it's an easy-to-scan task. 

### Resolves:
 [OKTA-159198](https://oktainc.atlassian.net/browse/OKTA-159198)

### CODE REVIEW

File: _source/_authentication-guide/saml-login/index.md [hunk 1]
```diff
@@ -10,64 +10,92 @@ redirect_from:
 
 ## Overview
 
-Okta supports authentication with an external SAML Identity Provider (IdP) that then redirects you to Okta and/or your application, which we call Inbound SAML. Additionally, it is possible to include an OpenID Connect (OIDC) ID token along with this redirect. This means that you can use Okta to proxy between SAML-only Identity Providers and OIDC-only applications that would normally be incompatible.
+Okta supports authentication with an external SAML Identity Provider (IdP). The SAML IdP redirects you to Okta or your application, also called Inbound SAML. You can also include an OpenID Connect ID token with this redirect. Thus, you can use Okta to proxy between SAML-only Identity Providers and OpenID Connect-only applications that normally are incompatible.
```
Comment (mystiberry-okta) [inline _source/_authentication-guide/saml-login/index.md]: Changes related to adding the algo start at line 25. The rest of these changes are simply to match our style guide and make this a true task topic.

File: _source/_authentication-guide/saml-login/index.md [hunk 2]
```diff
@@ -10,64 +10,92 @@ redirect_from:
...
+* An Okta OpenID Connect client application in the Okta org. If you don't have one already, you can add a client app using [the OpenID Connect Application Wizard](https://help.okta.com/en/prev/Content/Topics/Apps/Apps_App_Integration_Wizard.htm). 
 
-In addition to a SAML IdP that supports SP-initiated SAML, and an Okta org, this flow requires:
+2. Verify that your SAML IdP uses a supported algorithm:
+    * Digest
+        * <http://www.w3.org/2000/09/xmldsig#sha1>
+        * <http://www.w3.org/2001/04/xmlenc#sha256>
+    * Signature
+        * <http://www.w3.org/2000/09/xmldsig#rsa-sha1>
+        * <http://www.w3.org/2001/04/xmldsig-more#rsa-sha256>
```
Comment (mystiberry-okta) [inline _source/_authentication-guide/saml-login/index.md]: @tthompson-okta , the link on line 31 goes to a dead end. Is there a better link?

File: _source/_authentication-guide/saml-login/index.md [hunk 3]
```diff
@@ -10,64 +10,92 @@ redirect_from:
...
+    1. Select **Applications**.
+    2. Select the application you wish to modify.
+    3. Select **Assign** and choose  **Assign to Groups**.
+    4. Search for the appropriate group name. If this app is already assigned to the group, the name doesn't appear in the search box.
 
-### 3. Create an Authorize URL
+2. Select **General** and copy the **Client ID** (`client_id`) of the application for use in the next step.
 
-At this point you should have:
+## Create the Authorization URL
```
Comment (mystiberry-okta) [inline _source/_authentication-guide/saml-login/index.md]: Should this be Authentication URL? (here and elsewhere in this topic)?

**Comment by cwu-okta** [comment]
@royalchan-okta

File: _source/_authentication-guide/saml-login/index.md (hunk 2 above)
Comment (cwu-okta) [reply _source/_authentication-guide/saml-login/index.md]: Line 30 and 31 are both dead links. Should we use https://www.ietf.org/rfc/rfc4051.txt instead?

Comment (royalchan-okta) [reply _source/_authentication-guide/saml-login/index.md]: these URIs are the actual identifiers for the respective algorithms. They "work" in the sense that w3 redirects you to the correct xml file eventually:

image

Comment (tthompson-okta) [reply _source/_authentication-guide/saml-login/index.md]: Yea, we need to highlight these urls as "code" and not hyperlinks

File: _source/_authentication-guide/saml-login/index.md (hunk 1 above)
Comment (mystiberry-okta) [reply _source/_authentication-guide/saml-login/index.md]: Changes were removed as they didn't belong to this topic.

**Pull request merged**